bdpy/dl/torch/models.py

```diff
@@ -367,3 +367,447 @@ def forward(self, z: torch.Tensor) -> torch.Tensor:
         g = self.deconv(f)
 
         return g
+
+
+class AlexNetPool5Generator(nn.Module):
```

Comment:
Since many similar codes appear with the same responsibilities, it seemed more prospective to realize all classes added in this PR using one standardized implementation.